src/proto/NEAR.proto

```diff
@@ -0,0 +1,92 @@
+syntax = "proto3";
+
+package TW.NEAR.Proto;
+option java_package = "wallet.core.jni.proto";
+
+message PublicKey {
+    uint32 key_type = 1;
+    bytes data = 2;
+}
+
+message FunctionCallPermission {
+    bytes allowance = 1; // uint128
+    string receiver_id = 2;
+}
+
+message FullAccessPermission {
```

Comment:
this is empty message

Reply:
this is intentional, needed for future extension possibility